bin/brew

```diff
@@ -45,5 +85,77 @@ fi
 
 export HOMEBREW_BREW_FILE
 export HOMEBREW_RUBY_PATH
+export HOMEBREW_PREFIX
+export HOMEBREW_REPOSITORY
+export HOMEBREW_LIBRARY
+export HOMEBREW_CELLAR
+
+if [ -n "$HOMEBREW_OSX" ]
+then
+  if [ "$(xcode-\select --print-path)" = "/" ]
+  then
+    cat >&2 <<EOS
```

Comment:
I'd suggest to replace these instances with `odie` as well, also to get rid of the `cat` call. There are two possible options. Either we keep `odie` simple as proposed further above and pass a string argument instead of a heredoc, which would look something like:

```shell
odie "\
Error text …
… across multiple lines."
```

If this looks too weird, we can keep the heredoc and extend `odie` to accept the message via arguments and `stdin` like that:

```shell
odie() {
  local message
  if [[ $# -ne 0 ]] ; then
    message="$*"$'\n'
  else
    IFS= read -rd '' message
  fi
  printf "%s" "$message" >&2
  exit 1
}

odie <<EOS
Error text …
… across multiple lines.
EOS
```